index.html

```diff
@@ -28,180 +34,240 @@ <h1 class="header">I'am Sentayhu <br>  Glad to see <br>you!</h1>
+    <div class="desktop-hidden"></div>
+    <h2 class="tonic1">Tonic</h2>
+<div class="mobile-hidden tonic2">Facebook 360</div>
+ <br>
+    <div class="social-mv">
+     <ul class="connect-via card-info desktop-mv mobile-mv">
          <li class="canopy">Canopy </li>
      <li class="list1"><img src="./assets/images/Counter.svg" class="counters" alt="counters"> &nbsp;&nbsp; Back End Dev</li>
-     <li class="list1"><img src="./assets/images/Counter.svg" class="counters" alt="counters">&nbsp; &nbsp;2015</li>
-     </ul>
+     <li class="list1"><img src="./assets/images/Counter.svg" class="counters" alt="counters"> &nbsp; &nbsp; 2015</li>
+     </ul></div>
      <div class="discription">
        <p class="discpara">A daily selection of 
            privately personalized reads; no
             accounts or sign-ups required.</p>
      </div>
-     <ul class="connect-via">
+     <div class="social-mv">
+     <ul class="connect-via desktop-mv-lang">
          <li class="lang">HTML</li>
          <li class="lang">CSS</li>
          <li class="lang">JavaScript</li>
-     </ul>
+     </ul> </div>
      <button type="button" class="project1-btn">See Project</button>
     </div>
 </div>
+</div>
 <!-- ** END OF CARD 3 **-->
 <div class="subwork">
-    <div class="worksec1">
+    <div class="worksec1 worksec1d">
+        <div class="worksf" >
     <img src="./assets/images/Snapshoot%20Portfolio3.svg" class="cardpic" alt="pic">
-    <div class="card"></div>
-    <h1 class="tonic1">Multi-Post Stories</h1> 
-     <ul class="connect-via card-info">
+    
+</div>
+<!-- work sec start******** -->
+<div class="worksf">
+    <div class="mobile-hidden tonic2">Uber Navigation</div>
+    <div class="desktop-hidden">
+        <h2 class="tonic1">Multi-Post Stories</h2></div>
+    <div class="social-mv">
+     <ul class="connect-via card-info desktop-mv mobile-mv-1">
          <li class="canopy">Canopy </li>
      <li class="list1"><img src="./assets/images/Counter.svg" class="counters" alt="counters"> &nbsp;&nbsp; Back End Dev</li>
-     <li class="list1"><img src="./assets/images/Counter.svg" class="counters" alt="counters">&nbsp; &nbsp;2015</li>
-     </ul>
+     <li class="list1"><img src="./assets/images/Counter.svg" class="counters" alt="counters"> &nbsp; &nbsp; 2015</li>
+     </ul></div>
      <div class="discription">
        <p class="discpara">A daily selection of 
            privately personalized reads; no
             accounts or sign-ups required.</p>
      </div>
-     <ul class="connect-via">
+     <div class="social-mv">
+     <ul class="connect-via desktop-mv-lang">
          <li class="lang">HTML</li>
          <li class="lang">CSS</li>
          <li class="lang">JavaScript</li>
-     </ul>
+     </ul> </div>
      <button type="button" class="project1-btn">See Project</button>
     </div>
+    <div class="worksf">
+        <img src="./assets/images/Snapshoot%20Portfolio1.svg" class="cardpic1" alt="pic">
+    </div>
+</div>
 </div>
 </section>
```

Comment:
- [ ] Your card images have gone out of place since your last review. Please kindly adjust so it matches the original design.

### Your Design
![image](https://user-images.githubusercontent.com/43995830/154554529-041b7d6f-352a-4ad2-974f-923462dfd304.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/154554638-75774eaa-1976-4b01-8f0d-694a59368871.png)

Reply:
Hello @AnselemOdims ,
The card images does not go out of place on my laptop screen which I used to design kindly

Reply:
- [ ] dear @sentayhu19 , As underlined by the previous reviewer, your cards images went out of their place since the last review. Kindly fix them 🧑‍🔧 .  Look below how your template looks now for some screen size in the desktop version :

![image](https://user-images.githubusercontent.com/71874425/154569862-862fb378-0733-4d6c-8bbc-fbc5f88e4ef3.png)